src/renderer/ui/components/settings/ToggleableOption.js

```diff
@@ -26,6 +27,10 @@ class SettingsCheckbox extends Component {
           : null}
       </span>
     );
+    const dependsOnSettingsKey = this.props.dependsOnSettingsKey;
+    if (dependsOnSettingsKey && !this.props[dependsOnSettingsKey]) {
+      return <noscript />;
```

Comment:
You can return null from a component to render nothing, more optimal than mounting an unused DOM element.

`return null;`